scripts/wait-on-temporal.mjs

```diff
@@ -11,8 +11,11 @@ class Timer {
   }
 }
 
-async function main(maxAttempts = 100, retryIntervalSecs = 1) {
-  for (let attempt = 1; attempt <= maxAttempts; ++attempt) {
+const MAX_ATTEMPTS = 100,
+  RETRY_INTERVAL_SECS = 1;
```

Comment:
I haven't seen this syntax in a while :)

Reply:
all caps for finals, or multiple declarations in a single statement?

Reply:
my Java roots are showing 😄